docs/pages/modules/autolinking.mdx

```diff
@@ -204,9 +204,10 @@ All projects created with the `npx create-expo-app` command are already configur
 The module resolution algorithm searches only for packages that contain the [Expo module config](/modules/module-config/) file (**expo-module.config.json**) at the root directory, next to the **package.json** file.
 It's also necessary to include supported platforms in the `platforms` array — if the platform for which the autolinking algorithm is run is not present in this array, it's just skipped in the search results.
 
-### How is it different from React Native autolinking?
+### How is it different from React Native community CLI autolinking?
 
 - Expo Autolinking comes in with built-in support for monorepos, Yarn workspaces and transitive dependencies.
 - It's also significantly faster, even though the module resolution algorithm is more complex to be more reliable and match the Node's module resolution.
 - Expo module resolution is also capable of detecting the duplicates which is a common issue in monorepos.
 - Last but not least, it integrates well with all the features offered by Expo Modules APIs.
+- If you would like to use the React Native community CLI autolinking instead of Expo's Autolinking set `EXPO_USE_COMMUNITY_AUTOLINKING=1` and add `@react-native-community/cli` as a dev dependency to your project.
```

Comment:
```suggestion
- If you would like to use the React Native community CLI autolinking instead of Expo's Autolinking, set `EXPO_USE_COMMUNITY_AUTOLINKING=1` and add `@react-native-community/cli` as a dev dependency to your project.
```

One tiny nit that slipped me before.